release/fv/fv_ojibwa_ifinal/source/help/fv_ojibwa_ifinal.php

```diff
@@ -0,0 +1,234 @@
+                <td align='center'>ᣛ</td>
+                <td align='center'>ᐞ</td>
+            </tr>
+            <tr>
+                <td align='center'>123</td>
+                <td align='center'>🌐</td>
+                <td align='center'>ᐧ</td>
+                <td align='center' colspan='5'>space</td>
+                <th align='center'>˙</th>
+                <td align='center'>⌫</td>
+            </tr>
+
+        </table>
+        <h3>Shifted</h3>
+        <table class="keyboard-layer">
+            <tr>
+                <td align='center'>ᐁ</td>
+                <td align='center'>ᐯ ᕓ</td>
+                <td align='center'>ᑌ ᕞ</td>
+                <td align='center'>ᑫ</td>
+                <td align='center'>ᒉ</td>
+                <td align='center'>ᒣ</td>
+                <td align='center'>ᓀ ᓓ ᕃ</td>
+                <td align='center'>ᓭ</td>
+                <td align='center'>ᔐ</td>
+                <td align='center'>ᔦ</td>
+            </tr>
+            <tr>
+                <td align='center'>ᐅ</td>
+                <td align='center'>ᐳ ᕗ</td>
+                <td align='center'>ᑐ ᕤ</td>
+                <td align='center'>ᑯ</td>
+                <td align='center'>ᒍ ᓗ ᕈ</td>
+                <td align='center'>ᒧ</td>
+                <td align='center'>ᓄ</td>
+                <td align='center'>ᓱ</td>
+                <td align='center'>ᔓ</td>
+                <td align='center'>ᔪ</td>
+            </tr>
+            <tr>
+                <td align='center'>⇧</td>
+                <td align='center'>ᓬ ᓫ</td>
+                <td align='center'>ᕒ ᕑ</td>
+                <td align='center'>ᕃ ᕆ ᕊ ᕍ ᔆ</td>
+                <td align='center'>ᖊ ᖋ ᖌ ᖍ ᙆ</td>
+                <td>&nbsp;</td>
+                <td align='center'>ᐤ</td>
+                <td align='center'>ᐦ</td>
+                <td align='center'>ᣞ</td>
+                <td align='center'>ᣜ</td>
+            </tr>
+            <tr>
+                <td align='center'>123</td>
+                <td align='center'>🌐</td>
+                <td align='center'>ᐧ</td>
+                <td align='center' colspan='4'>space</td>
+                <td align='center'>᙮</td>
+                <td align='center'>⏎</td>
+                <td align='center'>⌫</td>
+            </tr>
+        </table>
+        <div class='vspace'></div>
+        <ul>
+            <li>keys with a red background are combining accents. On your mobile device, they will have green
+                text.</li>
+            <li>the first character in a cell is the "one-tap" key, any further characters are "hold-select"
+                keys.</li>
+            <li>Common punctuation can be found by pressing and holding the full stop (᙮) key on the right side
+                of the
+                space bar. </li>
+        </ul>
+        <div class='vspace'></div>
+    </div>
+    <div class='vspace'></div>
+
+    <br />
+
+    <h2>Keyboard Layout</h2>
+    <div id='osk'></div>
```

Comment:
This is how you can display all the layers for the desktop and mobile layouts. There may be more - this is what I saw on first glance

```suggestion
    <h2>Keyboard Layout</h2>
    <h3>Desktop Layout</h3>
    <div id='osk' data-states='default shift rightalt rightalt-shift'></div>

    <h3>Mobile Layout</h3>
    <div id='osk-phone' data-states='default shift numeric'></div>
```

Reply:
I'll add the desktop ones in. We made the decision to make tables for the mobile keyboards instead of including the OSK images so that users can see options that available in long presses. We're working on making that standard across all the keyboards within our scope.